### ISSUE TITLE
Error when installing (while loading init.el)

### ISSUE THREAD

**Issue by manuporto** [syl20bnr/spacemacs#3361]
I'm having the following problem when I try to install Spacemacs. My emacs version es 24.5. Here is the error:
```
Warining (initialization): An error occurred while loading `/home/xxxxx/.emacs.d/init.el`:

Wrong type argument: (or eieio-object-p class-p), nil
```

I tried removing the .emacs.d directory and installing again but it didn't work. I also tried reinstalling emacs but I was also out of luck.

**Comment by robbyoconnor** [comment]
@manuporto `SPC hds` please if you can. 


If you cannot: Are you using `develop` or `master`?

**Comment by robbyoconnor** [comment]
If using `master` -- please try using `develop` to see if the issue resolves itself.

**Comment by manuporto** [comment]
What do you mean by `SPC hds`? 

I am using `master`, but I just tried with `develop` and the error persists. My os is Ubuntu 14.04

**Comment by robbyoconnor** [comment]
What version of spacemacs?

**Comment by robbyoconnor** [comment]
Try with a fresh `.spacemacs` file

**Comment by manuporto** [comment]
@robbyoconnor deleting the `.spacemacs` file and running Emacs again worked. The version I am using is  0.104.2

Thank you!

**Comment by Migacz85** [comment]
-Fresh installation and I have something like that : 
![image](https://user-images.githubusercontent.com/25789997/62887153-9f4bba00-bd34-11e9-8086-83a523ac7ac4.png)

![image](https://user-images.githubusercontent.com/25789997/62887271-d3bf7600-bd34-11e9-9104-df77bd9e26c6.png)

What is going on ?

**Comment by duianto** [comment]
It's most likely this issue:
Failed to download 'gnu' archive. Bad request. (Solution included) #12535

It's an issue with Emacs, the linked bugreport in that issue has marked the issue as fixed in the next Emacs version 26.3.

You should be fine installing with `--insecure`, it just uses http instead of https.

But it's probably best to use the workaround (in that issue) going forward.

**Comment by Migacz85** [comment]
I also have this kind of errors:

![image](https://user-images.githubusercontent.com/25789997/63018906-da173480-be91-11e9-9275-0557a9cda21f.png)

**Comment by Migacz85** [comment]
I just want to have `stable` config to start with this editor. How I should properly resolve this issues?

**Comment by Migacz85** [comment]
```
;; Work around for Hunspell 1.7.0 
(defun manage-hunspell-1.7 (old-function-ispell &rest arguments)
  "Add null-device when calling \"hunspell -D\"."
  (if  (equal "-D"  (nth 4 arguments))
      (funcall old-function-ispell "hunspell" null-device t nil "-D" null-device)
    (apply old-function-ispell  arguments)))
(advice-add 'ispell-call-process :around #'manage-hunspell-1.7)

````
That's somehow helps. But still

**Comment by duianto** [comment]
The first error appeared when we added the new `helm-org` layer.

Another maintainer smile13241324 said this about it:
>Its the autoload of
```
(easy-menu-add-item
 nil '("Tools" "Helm")
 '("Org"
   ["Org headlines in org agenda files" helm-org-agenda-files-headings t]
   ["Org headlines in buffer" helm-org-in-buffer-headings t])
 "Elpa")
```
>which should actually use the local menu-bar keymap but it seems to be nil

I'm not sure how to fix it.

The No word lists message sounds like a dictionary issue, maybe your latest comment about Hunspell fixed it.

The ptf-v error seems like a space in a variable issue, the variable name without a space is:
`pdf-view-midnight-colors`
Found a mention of it here: https://emacs.stackexchange.com/questions/40745/recolouring-pdfs
